### ISSUE TITLE
Neko/Windows/Flash SA player: Kill processes before each new build + Reopen them after each new build

### ISSUE THREAD

**Issue by sruloart** [lucamezzalira/haxe-watchify#11]
Maybe with using this method: http://forum.thewindowsclub.com/windows-tips-tutorials-articles/29463-kill-processes-using-command-prompt-windows-7-8-a.html. 

This way you can rebuild them and actually see the results (because they aren't being currently in use) :)

it might also circumvent the FlashDevelop whole "can't delete these files" on export/windows/cpp/neko issue, but maybe not.

**Comment by lucamezzalira** [comment]
I didn't know that windows needs a different parameter to close the process, my bad:
http://stackoverflow.com/questions/10021373/what-is-the-windows-equivalent-of-process-onsigint-in-node-js

I'm going to test it tonight, hopefully tomorrow morning you'll wake up with a new version

**Comment by sruloart** [comment]
Not your fault, I blame Microsoft :)

**Comment by lucamezzalira** [comment]
eheheh that's the good of open source projects, everyday you learn something new :D

**Comment by lucamezzalira** [comment]
it should solved in the next release

**Comment by sruloart** [comment]
Cool, this would be a great release with all of these new features :D

**Comment by lucamezzalira** [comment]
please check if in the version 1.3.0 is fixed :+1:

**Comment by sruloart** [comment]
Yeah, I'm on Win10 now, and now these targets' windows don't open and don't close.

**Comment by lucamezzalira** [comment]
Ok I'll check it on win10, did you try to set the buildtype parameter as build and use the livereload to update the content?

**Comment by sruloart** [comment]
Yep.
```json
{
  "build": {
    "program" : "openfl",
    "buildType" : "build", 
	"src" : "source",
	"monitor" : "web",
    "livereload" : "export",
    "platforms" : ["flash", "neko", "windows"]
  }
}
```
